### DISCUSSION TITLE
Restarting Open Free Energy Calculations from Checkpoints on AWS Spot Instances

### DISCUSSION THREAD

**Discussion by stream0426** [OpenFreeEnergy/openfe#868] [Q&A]
Hi,

I am planning to perform Open Free Energy calculations (RBFE) using AWS Spot Instances. Given that jobs are interrupted when instances are stopped, I would like to set up a system where the job restarts from a checkpoint file if one exists.

Would it be possible to share any scripts you might have for this purpose? Additionally, if there are specific parts of the source code that should be modified to allow for resuming from checkpoint files (such as determining the presence of a checkpoint file), I would appreciate any guidance on that as well.

Thank you in advance for your assistance.

Best regards,
Hiroyuki

**Comment by IAlibay** [comment]
@stream0426 thanks for reaching out and apologies for the delayed response.

Unfortunately we currently do not support restarts at this moment in time. This is a feature we do want / are looking to impelement. We will update as soon as possible.

**Reply by stream0426** [reply]
Thank you for your response. I appreciate the transparency regarding the feature's current status and the team's intention to implement it in the future. Looking forward to any updates on this.
